packages/trpc/server/routers/viewer/dsync/delete.handler.ts

```diff
@@ -27,7 +27,7 @@ export const deleteHandler = async ({ ctx, input }: Options) => {
     });
   }
 
-  await prisma.dSyncData.delete({
+  await prisma.dSyncData.findMany({
```

Comment:
what is the user of findMany here ?

Reply:
Also not sure what `findMany` does here, then we don't delete anything? Would `deleteMany` solve it? that shouldn't throw an error if it doesn't exist

Reply:
The dsync data still exists in the SAML DB which is handled with the Jackson package.

Reply:
I was blanking so hard on this. It was supposed to be deleteMany